TheJumpCloud/support
# NB-11998: Windows App Launches After Install

## Issues
* [NB-11998](https://jumpcloud.atlassian.net/browse/NB-11998) - Windows app opening on launch after specifying $LaunchPasswordManager to $false

## Dependencies
- https://github.com/jc-pny/desktop-app-v2/pull/679

## What does this solve?
Windows app opening on launch after specifying $LaunchPasswordManager to $false

## Is there anything particularly tricky?
Yes, the testing

## How should this be tested?
1- Download the JumpCloud Password Manager `v2.2.189` `.exe` from [AWS](https://us-east-2.console.aws.amazon.com/s3/buckets/jc-pwm-builds-stg01?region=us-east-2&bucketType=general&prefix=DA/staging/release/&showversions=false)
2- Create the below `.ps1` script on your machine
```
$LaunchPasswordManager = $false
$installerTempLocation = "<PATH_TO_DOWNLOADED_APP_EXE>"
if ($LaunchPasswordManager -eq $false) {
    $env:QUIT_PWM_AFTER_INITIAL_INSTALL="true"
}
. $installerTempLocation
```
3- Replace `<PATH_TO_DOWNLOADED_APP_EXE>` with the path of the downloaded file
4- Run the script => The app should not launch after installation
5- Now open a new terminal tab (this is important)
6- Modify `$LaunchPasswordManager = $false` to `$LaunchPasswordManager = $true`
7- Run the script => The app should launch after installation

## Screenshots
N/A

[NB-11998]: https://jumpcloud.atlassian.net/browse/NB-11998?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
